plugins/server_logs/samson_slack/samson_plugin.rb

```diff
@@ -0,0 +1,18 @@
+require 'slack'
```

Comment:
probably can remove this line and put this in `server_logs/lib/server_logs` because the constructor for a `Plugin` goes up 3 directories